package.json

```diff
@@ -292,6 +292,7 @@
     "@metamask/approval-controller": "^7.0.0",
     "@metamask/assets-controllers": "^34.0.0",
     "@metamask/base-controller": "^5.0.1",
+    "@metamask/bitcoin-wallet-snap": "^0.2.1",
```

Comment:
Shouldn't this be `^0.2.2` to make sure we reference the version the event emitting?